Source/SwiftLintBuiltInRules/Rules/Style/ImplicitOptionalInitializationRule.swift

```diff
@@ -0,0 +1,160 @@
+import SwiftSyntax
+
+@SwiftSyntaxRule(explicitRewriter: true)
+struct ImplicitOptionalInitializationRule: Rule {
+    var configuration = ImplicitOptionalInitConfiguration()
+
+    static let description = RuleDescription(
+        identifier: "implicit_optional_initialization",
+        name: "Implicit Optional Initialization",
+        description: "Optionals should be consistently initialised",
+        kind: .style,
+        nonTriggeringExamples: ImplicitOptionalInitRuleExamples.nonTriggeringExamples,
+        triggeringExamples: ImplicitOptionalInitRuleExamples.triggeringExamples,
+        corrections: ImplicitOptionalInitRuleExamples.corrections,
+        deprecatedAliases: ["redundant_optional_initialization"]
+    )
+}
+
+private extension ImplicitOptionalInitializationRule {
+    final class Visitor: ViolationsSyntaxVisitor<ConfigurationType> {
+        override func visitPost(_ node: VariableDeclSyntax) {
+            violations.append(
+                contentsOf: node.violationPositions(for: configuration.style).map {
+                    ReasonedRuleViolation(position: $0, reason: reason)
+                })
+        }
+
+        var reason: String {
```

Comment:
Can we have this as a computed property of the visitor? That would make it easier to find.